### DISCUSSION TITLE
Beats port 5044

### DISCUSSION THREAD

**Discussion by LennyDK** [Security-Onion-Solutions/securityonion#2817] [Unsupported Versions]
Hi there
I seam to have a problem with opening the Beat port 5044 with so-allow on version 2.3.21
Despite running the so-allow script the manager server keeps dropping traffic from winlogbeat with a unauthorized answer in the packet on the server. And yes my added IP's are correct with a 192.168.50.0/24 network. In the iptables config i can see they do get added in there. 

I am running matching winlogbeat version to the server. And the log from winlogbeat just say i can't connect and traffic is dropped on the manager server. It is running in vmware workstation 16.1.x  

Is there something i am missing here ore do others have that same challenge ?

Best regards 
Lenny

**Comment by cm-ops** [comment]
@LennyDK 

Can you share the tail of the winlogbeat.log? Also, check your winlogbeat.yml for spacing/formatting and verify it is configured according to https://docs.securityonion.net/en/latest/beats.html?highlight=winlogbeat#winlogbeat.

**Reply by endpointsec** [reply]
Open a powershell prompt from your winlogbeat dir and perform the following:

.\winlogbeat.exe test config
.\winlogbeat.exe test output

If you want to see live output, I think this is it (rusty at the moment, been a while):
.\winlogbeat.exe -c .\winlogbeat.yaml -e

**Comment by nezvocowa** [comment]
Sounds more like a certificate problem since you got a `unauthorized answer` response? Have you checked the firewall logs to see if it actually drops/rejects the packets?

**Comment by LennyDK** [comment]
Hi there
There no firewalls in between other then security onion’s own

Venlig hilsen / Best Regards
Lenny Hansson
*****************************
Web: https://networkforensic.dk
*****************************
E-mail: Lenny@netcowboy.dk
Key ID: 5189 6702 3A0F 4C07 4AF5 5619 B6E0 D375 F0CB F2E8
*****************************

**Comment by LennyDK** [comment]
Thanks
I Will test this. 

Venlig hilsen / Best Regards
Lenny Hansson
*****************************
Web: https://networkforensic.dk
*****************************
E-mail: Lenny@netcowboy.dk
Key ID: 5189 6702 3A0F 4C07 4AF5 5619 B6E0 D375 F0CB F2E8
*****************************

**Comment by jcastillo725** [comment]
Hi LennyDK,

Have any luck on the issue? I'm experiencing the same, I had the IP range so-allowed and verified that the firewall is allowing the connection but I'm getting a message about a failed connection attempt.

Here's what I'm seeing in the winlogbeat logs:

> 2021-04-06T23:39:18.644+0800	ERROR	[publisher_pipeline_output]	pipeline/output.go:154	Failed to connect to backoff(async(tcp://192.168.1.40:5044)): dial tcp 192.168.1.40:5044: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
> 
> 2021-04-06T23:39:18.644+0800	INFO	[publisher_pipeline_output]	pipeline/output.go:145	Attempting to reconnect to backoff(async(tcp://192.168.1.40:5044)) with 1 reconnect attempt(s)
>

**Reply by defensivedepth** [reply]
@jcastillo725 Please start a new Discussion, this is an old discussion. Thanks!
